bigbluebutton-html5/imports/ui/components/presentation/service.js

```diff
@@ -55,11 +55,21 @@ const getCurrentSlide = (podId) => {
   });
 };
 
-const getSlidePosition = (podId, presentationId, slideId) => SlidePositions.findOne({
-  podId,
-  presentationId,
-  id: slideId,
-});
+const getSlidePosition = (podId, presentationId, slideId) => {
+  const positions = SlidePositions.findOne({
+    podId,
+    presentationId,
+    id: slideId,
+  });
+
+  if (!positions) {
+    return {
+      width: 1000, height: 1000, x: 0, y: 0, viewBoxWidth: 1000, viewBoxHeight: 1000,
```

Comment:
Hi @ramonlsouza 
Is there a reference or justification for using those specific values as default? Could they include other type of issues in the components?